WoWPro_Leveling/Classic/Alliance/12_20_Darkshore.lua

```diff
@@ -260,9 +260,14 @@ H Auberdine|ACTIVE|942|N|Hearth back to Auberdine or run to Astranaar and fly ba
 b Wetlands|QID|942|M|32.42,43.74|N|Get on the boat to Menethil Harbor (read the sign post).|
 f Menethil Harbor|QID|942|M|9.48,59.68|Z|Wetlands|N|Grab the flightpath.|TAXI|-Menethil Harbor|
 T The Absent Minded Prospector|QID|942|M|10.84,60.43|Z|Wetlands|N|To Archaeologist Flagongut, on the 2nd floor of the Deepwater Tavern. Don't take follow up, too high level for now.|
-N Heading to Stormwind|QID|968|N|From here we need to get to Ironforge and on to Stormwind City to grab the FPs for the other major cities and turn in a quest (if you found the book). If you do not already have means to get to Ironforge (Mage/Warlock/FP) then you can follow the next steps to get there relatively quickly |
-N I'm on a boat|QID|968|N| To get to IF quickly you can use the following steps. \nGet onto the boat to Theramore at the docks. \nAs it sets off look out on the port side, once you see the boat get very close to land jump off. \nOpen the map and look at the Dun Morogh map you will see your character in the north above the mountains. \nRun west along the coast until you arrive at the small bay in the top left visible on the Dun Morogh map. \nHead towards the shore in the bay until you see your General Chat change to Dun Morogh. \nWhilst staying in the Dun Morogh zone drown yourself in the water. \nFrom the graveyard head north and then east at the fork after the bridge. \nOnce inside Ironforge take a right and zone into the Deeprun tram. \nYou will end up in the Deeprun Tram alive again and can now head back into Ironforge|
-f Ironforge|QID|968|M|55.49,47.72|Z|Ironforge|N|Grab the flightpath from Gryth Thurden.|TAXI|-Ironforge|
+F Ironforge|QID|968|N|Fly to Ironforge|TAXI|Ironforge|
```

Comment:
For Travel or Note steps, use ACTIVE instead of QID if you are going to turn in the step, otherwise use AVAILABLE if it is leading to a quest that will be accepted.   So `P Moonglade|AVAILABLE|26` should be ACTIVE since it is followed by a T step with that QID.  All the travel steps that lead to `T The Powers Below|QID|968|` should use ACTIVE.  Although the `F Rut'theran Village|AVAILABLE|26|` will work, it is better to use `ACTIVE| 6125` as it is the closest enclosing quest